src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/x64/reduce.cpp

```diff
@@ -699,7 +828,21 @@ INSTANTIATE_TEST_SUITE_P(
         ReduceCPULayerTest::getTestCaseName
 );
 
+INSTANTIATE_TEST_SUITE_P(
+        smoke_Reduce_DimZero_Arithmetic_fusing_KeepNoDims_CPU,
+        ReduceCPULayerTest,
+        params_DimZero_Arithmetic_fusing_KeepNoDims,
+        ReduceCPULayerTest::getTestCaseName
+);
+
+INSTANTIATE_TEST_SUITE_P(
```

Comment:
Any reason to have multiple test suites to cover empty input case? Can we have one test suite dedicated to cover this problem?

Reply:
Test suites of KeepNoDims are removed, as the value of keep_dim to be true or false won't change the result of this case. Besides, I've merged non-fusion test suites into fusion test suites. So there are only two test suites of empty input left now. 

The reason that I split the test suite into Algorithm and Compare(ReduceMax/Min) is that ReduceMax/Min are tested only on infer_precision=f32. The reason is, e.g. f16 for empty input, ReduceMax will return f16-lowest, while the expected test output is f32-lowest. Though there is a Convert node inserted to convert f16->f32, the value itself won't change. I think it doesn't make much sense to support conversion from f16-lowest to f32-lowest etc in CPU Convert node/emitters. Please feel free to have further discussion if you have concerns! Thanks Dmitry!